libs/trpc/src/lib/context.ts

```diff
@@ -1,12 +1,39 @@
 import * as trpcExpress from '@trpc/server/adapters/express';
 
 import { inferAsyncReturnType } from '@trpc/server';
+import { prisma } from '@dnd-assistant/prisma';
 
-export const createContext = (
+export const createContext = async (
   expressContext: trpcExpress.CreateExpressContextOptions
 ) => {
-  console.log('Request headers', expressContext.req.headers);
-  return {};
+  const innerContext = await getInnerContext(expressContext);
+  const { req, res } = expressContext;
+  return {
+    ...innerContext,
+    req,
+    res,
+  };
+};
+
+const getInnerContext = async (
+  expressContext: trpcExpress.CreateExpressContextOptions
+) => {
+  const session = await getSessionFromAuthHeader(
+    expressContext.req.headers.authorization
+  );
+  return {
+    session,
+  };
+};
```

Comment:
This function could be completely removed and it's behavior put into `createContext` since it adds no value

Reply:
Sure I'm okay with removing, was following trpc [recommendation ](https://trpc.io/docs/server/context#inner-and-outer-context)of inner and outer contexts, but it doesn't feel like our application has anything more that's needed for an inner context than session

Reply:
Huh, I find that example super odd even in their proposed form... either way, good with moving forward with it and going to mark as resolved